src/Cmgmyr/Messenger/examples/MessagesController.php

```diff
@@ -30,7 +30,9 @@ public function index()
         // All threads that user is participating in, with new messages
         // $threads = Thread::forUserWithNewMessages(Auth::id())->latest('updated_at')->get();
 
-        return view('messenger.index', compact('threads'));
```

Comment:
Using `view` helper is Laravel way already: https://laravel.com/docs/5.4/blade#displaying-data :}